fe/fe-core/src/main/java/com/starrocks/common/util/Daemon.java

```diff
@@ -92,7 +93,7 @@ public void setInterval(long intervalMs) {
     /**
      * implement in child
      */
-    protected void runOneCycle() {
+    protected void runOneCycle() throws AnalysisException {
```

Comment:
`AnalysisException` is deprecated, and why we need to throw exception here?

Reply:
I have replaced AnalysisException with SemanticException, so throwing the exception is no longer necessary.